In [29]:
!dir

 Volume in drive C has no label.
 Volume Serial Number is 1C2B-1A37

 Directory of c:\Users\fabian\Desktop\Facultad FINAL BOSS\Deep-Learning\Practica\Practica0

20/08/2023  12:31    <DIR>          .
20/08/2023  12:31    <DIR>          ..
20/08/2023  12:54             7.554 ejercicio2.ipynb
15/08/2023  23:02           278.288 Pr�ctica 0 - Introducci�n a Python.pdf
               2 File(s)        285.842 bytes
               2 Dir(s)  196.385.902.592 bytes free


El comando `!dir` en Windows te muestra el contenido del directorio actual. En el resultado que proporcionaste, estás viendo el contenido del directorio `c:\Users\fabian\Desktop\Facultad FINAL BOSS\Deep-Learning\Practica\Practica0`.

Aquí hay un desglose de lo que muestra el resultado:

1. **Volume in drive C has no label.**: Indica que el volumen del disco C no tiene una etiqueta asignada.
2. **Volume Serial Number is 1C2B-1A37**: Es el número de serie del volumen del disco.
3. **Directory of c:\Users\fabian\Desktop\Facultad FINAL BOSS\Deep-Learning\Practica\Practica0**: Es la ruta del directorio que estás viendo.
4. **...**: Indica que hay más archivos o directorios anteriores que no se muestran en este fragmento.
5. **20/08/2023  12:37               967 ejercicio2.ipynb**: Es un archivo Jupyter Notebook llamado "ejercicio2.ipynb" con un tamaño de 967 bytes y fue modificado por última vez el 20 de agosto de 2023 a las 12:37.
6. **15/08/2023  23:02           278.288 Pr�ctica 0 - Introducci�n a Python.pdf**: Es un archivo PDF (posiblemente con un nombre que contiene caracteres especiales que no se muestran correctamente) con un tamaño de 278,288 bytes y fue modificado por última vez el 15 de agosto de 2023 a las 23:02.
7. **2 File(s)        279.255 bytes**: Indica que hay 2 archivos en total en este directorio con un tamaño combinado de 279,255 bytes.
8. **2 Dir(s)  203.967.533.056 bytes free**: Indica que hay 2 directorios (o carpetas) en este directorio y que hay 203,967,533,056 bytes (aproximadamente 203.9 GB) de espacio libre en el disco C.

¿Hay algo específico que te gustaría saber o hacer con esta información?

In [30]:
!dir /b

ejercicio2.ipynb
Pr�ctica 0 - Introducci�n a Python.pdf


In [31]:
!python --version

Python 3.11.4


In [32]:
!pip show pandas

Name: pandas
Version: 2.0.3
Summary: Powerful data structures for data analysis, time series, and statistics
Home-page: 
Author: 
Author-email: The Pandas Development Team <pandas-dev@python.org>
License: BSD 3-Clause License

Copyright (c) 2008-2011, AQR Capital Management, LLC, Lambda Foundry, Inc. and PyData Development Team
All rights reserved.

Copyright (c) 2011-2023, Open source contributors.

Redistribution and use in source and binary forms, with or without
modification, are permitted provided that the following conditions are met:

* Redistributions of source code must retain the above copyright notice, this
  list of conditions and the following disclaimer.

* Redistributions in binary form must reproduce the above copyright notice,
  this list of conditions and the following disclaimer in the documentation
  and/or other materials provided with the distribution.

* Neither the name of the copyright holder nor the names of its
  contributors may be used to endorse or promote

In [33]:
!pip show numpy

Name: numpy
Version: 1.25.2
Summary: Fundamental package for array computing in Python
Home-page: https://www.numpy.org
Author: Travis E. Oliphant et al.
Author-email: 
License: BSD-3-Clause
Location: C:\Users\fabian\AppData\Local\Programs\Python\Python311\Lib\site-packages
Requires: 
Required-by: pandas


Creamos una carpeta como ejemplo

In [34]:
!mkdir test_folder

Eliminamos esa carpeta

In [35]:
!rmdir test_folder